libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -1469,19 +1472,60 @@ private static void addVideoUploadingClassIfMissing(AztecAttributes attributes)
         attributes.setValue(ATTR_CLASS, attrs.getAttributes().getValue(ATTR_CLASS));
     }
 
+    private static Attributes getFirstElementAttributes(Spanned content, AztecText.AttributePredicate predicate) {
+        List<Attributes> firstAttrs = getElementAttributes(content, predicate, true);
+        if (firstAttrs.size() == 1) {
+            return firstAttrs.get(0);
+        }
+        else {
+            return null;
+        }
+    }
+
+    private static @NonNull List<Attributes> getAllElementAttributes(Spanned content,
+                                                                  AztecText.AttributePredicate predicate) {
+        return getElementAttributes(content, predicate, false);
+    }
+
+    private static @NonNull List<Attributes> getElementAttributes(Spanned content,
```

Comment:
Duplicating `getElementAttributes()` and `updateElementAttributes()` from `AztecText` feels a bit ugly - I think it works for now, but we should probably look at tucking these back into the Aztec library at some point.